### ISSUE TITLE
can this be used to super impose an image on top?

### ISSUE THREAD

**Issue by pke** [ezartech/ezar-facedetector#1]
Instead of rectangles I'd like to superimpose a png over the face. Would that be possible?

**Comment by ezartech** [comment]
Absolutely! Simply position and size your face overlay image(s) over the rectangle(s) of the faces detected by the ezAR FaceDetector plugin.

**Comment by azuspace** [comment]
Hi ezAR! I'm superimposing images over the rectangle of the faces detected by the ezAR facedetector plugin. It works great in Android but in iOS the overlay images are displaced towards the left or right. This is only happening for the FRONT camera position while the BACK camera position doesn't have any displacement in the rendered image. Any help that you can provide would be greatly appreciated!
